subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -447,28 +447,55 @@ void *radio_pkt_decrypt_get(void)
 #endif
 
 #if !defined(CONFIG_BT_CTLR_TIFS_HW)
+
+#define SW_SWITCH_PREV_RX             0
+#define SW_SWITCH_NEXT_RX             0
+#define SW_SWITCH_PREV_TX             1
+#define SW_SWITCH_NEXT_TX             1
+#define SW_SWITCH_PREV_PHY_1M         0
+#define SW_SWITCH_PREV_FLAGS_DONTCARE 0
+#define SW_SWITCH_NEXT_FLAGS_DONTCARE 0
```

Comment:
could be defined in `pdu.h` and eventually also used in LLL since flags are magic values there as well

Reply:
Again the macros are replacement to hardcoded values passed to `sw_switch` function which is vendor specific interface, not sure if other vendors will use `0`  as the dont care values.

Change other magic values is outside the scope of this PR.